client/src/RatingsAndReviews/Reviews/Reviews.jsx

```diff
@@ -110,14 +113,22 @@ class Reviews extends React.Component {
             viewPhoto={this.viewPhoto}/>
           <div className="review-buttons">
             <button
+              interaction="loaded more reviews"
               className="review-button"
-              onClick={this.loadTwoMore}
+              onClick={(e) => {
+                this.loadTwoMore(e);
+                reviewsInteraction(e);
```

Comment:
same idea here - replace reviewsInteraction with renderProps function after its created 👌 just to give a couple examples